### ISSUE TITLE
Includes jQuery Dependencies and scripts

### ISSUE THREAD

**Issue by capellopablo** [htmlburger/wpemerge#18]
Hello everyone.
I am trying to work with JS on my project.

Following this guide I was able to create my own JS files: https://docs.wpemerge.com/#/starter-theme/assets/overview?id=importing-assets

But .. Now I want to include a jQuery dependency.
For example. I need to include Owl Carousel.

I installed it with yarn add.
But ... how should I import it and use it in my JS files? (scripts/resources/theme)

Thanks!

**Comment by atanas-angelov-dev** [comment]
Hi @capellopablo ,

The usage should be exactly as Owl Carousel's docs state:
```js
// other imports go here.
import 'owl.carousel/dist/assets/owl.carousel.css';
import 'owl.carousel';

$(document).ready(function(){
  $('.owl-carousel').owlCarousel();
});
```

**Comment by capellopablo** [comment]
Hi @atanas-angelov-dev, thanks for your time.

Well, I could import js.
I installed Owl Carousel 2, So i import it with `import 'owl.carousel2';` in main index.js.

But I cant import CSS.
I tried import it doing: 

```
// In index.js
import 'owl.carousel2/dist/assets/owl.carousel.css';
```
But yarn build return:
```
ERROR in ./node_modules/owl.carousel2/dist/assets/owl.carousel.css
Module build failed (from ./node_modules/extract-text-webpack-plugin/dist/loader.js):
ModuleParseError: Module parse failed: Unexpected character (1:0)
You may need an appropriate loader to handle this file type.
(Source code omitted for this binary file)

ERROR in ./node_modules/owl.carousel2/dist/assets/owl.video.play.png 1:0
Module parse failed: Unexpected character (1:0)
You may need an appropriate loader to handle this file type.
(Source code omitted for this binary file)
 @ ./node_modules/owl.carousel2/dist/assets/owl.carousel.css 7:2648-2679
```

I also tried import it from index.scss, but I failed.

Finally I tried import scss:
`@import '~owl.carousel2/src/scss/owl.carousel';`
But it didn't work either.

```
ERROR in ./resources/styles/theme/index.scss
Module build failed (from ./node_modules/extract-text-webpack-plugin/dist/loader.js):
ModuleNotFoundError: Module not found: Error: Can't resolve './owl.video.play.png' in '/Users/capellopablo/dev/local/codersclan-projects/ylventures/app/public/wp-content/themes/airfleet/resources/styles/theme'
```

I'm sorry, I'm a little confused.

**Comment by atanas-angelov-dev** [comment]
It seems the webpack image config is a bit restricting. Here's how to get it working:
`resources/build/lib/utils.js`
Change this line:
`images: /images[\\/].*\.(ico|jpg|jpeg|png|svg|gif)$/,`
to this:
`images: /(images|node_modules)[\\/].*\.(ico|jpg|jpeg|png|svg|gif)$/,`

`resources/scripts/theme/index.js`
```js
import 'owl.carousel2/dist/assets/owl.carousel.css';
import 'owl.carousel2/dist/assets/owl.theme.default.css';
import 'owl.carousel2';

$(document).ready(function(){
  $('.owl-carousel').owlCarousel();
});
```

I'll be looking into this so the issue is resolved in the next version.

**Comment by capellopablo** [comment]
Really thanks @atanas-angelov-dev.
This has worked for me.

**Comment by atanas-angelov-dev** [comment]
Fixed in htmlburger/wpemerge-theme@706aea5c027756bd5e590aa0f0a7e4ba5dab223c
